### DISCUSSION TITLE
Hide Chain Selector

### DISCUSSION THREAD

**Discussion by garmeeh** [rainbow-me/rainbowkit#442] [Q&A] [answered]
Is there a way to hide the chain selector?

It hides by default when only one chain is provided so maybe this could be exposed as a prop?

**Comment by markdalgleish** [comment, marked as answer]
Try this: `<ConnectButton chainStatus="none" />`

Here's some more detail about the props ConnectButton supports: https://www.rainbowkit.com/docs/connect-button#props.
